build.gradle

```diff
@@ -43,12 +23,12 @@ ext {
 
 def dependencyVersions = [
         opensaml              : "$opensaml_version",
-        ida_utils             : '2.0.0-392',
+        ida_utils             : '2.0.0-395',
         dropwizard            : '1.3.8',
-        ida_dev_pki           : '1.1.0-37',
-        saml_libs             : "$opensaml_version-250",
-        ida_test_utils        : '2.0.0-46',
-        hub_saml              : "$opensaml_version-15766",
+        verify_dev_pki        : '2.0.0-45',
+        saml_libs             : "$opensaml_version-254",
```

Comment:
https://repo1.maven.org/maven2/uk/gov/ida/saml-lib/ version 255?

Reply:
That latest version of saml-libs was a downgrade to compile it to Java8 byte code. That was so that the MSA and VSP could use it. Version 254 is compiled to Java11 bytecode. We run the stub-idp with Java 11 so it's ok to stay on this version for now.